src/Helper/Url.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Findologic\Plentymarkets\Helper;
+
+class Url
+{
+    public static function getHost(string $url): string
+    {
+        $parseUrl = parse_url(trim($url));
+
+        if (!empty($parseUrl['host'])) {
```

Comment:
When is the `host` empty and when do we need to manually parse the host by dropping everything after `/`?

Reply:
If `$url` does not have protocol (`www.example.com`), `parse_url` returns an associative array that does not have `host` value. If protocol is set, then `host` value is set.